app/services/analytics_events.rb

```diff
@@ -2885,7 +2996,14 @@ def idv_personal_key_visited(
   # @param [Hash] errors Errors resulting from form validation
   # @param [Hash] error_details Details for errors that occurred in unsuccessful submission
   # @param ["sms", "voice"] otp_delivery_preference
-  # @param [Idv::ProofingComponentsLogging] proofing_components User's current proofing components
+  # @param [Hash,nil] proofing_components User's current proofing components
+  # @option proofing_components [String,nil] document_check Vendor that verified the user's identity documents
```

Comment:
https://rubydoc.info/gems/yard/file/docs/Tags.md#option

```suggestion
  # @option proofing_components [String,nil] :document_check Vendor that verified the user's identity documents
```
the option should be the string or symbol for these